src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -307,7 +307,7 @@ Modelers:
     <value>Errors found during Swagger document validation.</value>
   </data>
   <data name="XMSPathBaseNotInPaths" xml:space="preserve">
-    <value>Paths in x-ms-paths must overload a normal path in the paths section.</value>
+    <value>Paths in x-ms-paths must overload a normal path in the paths section, i.e. a path in the x-ms-path must either be same as a path in the paths section or a path in the paths sections followed by additional parameters.</value>
```

Comment:
@veronicagg  Here is the sample error message:

ERROR: Paths in x-ms-paths must overload a normal path in the paths section, i.e. a path in the x-ms-paths must either be same as a path in the paths section or a path in the paths sections followed by additional parameters.
        Path: file:///C:/workspace/azure-rest-api-specs/arm-apimanagement/2016-07-07/swagger/apimanagement.json#/x-ms-paths/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiX}?export=true